src/output/Thread.cxx

```diff
@@ -267,7 +267,6 @@ AudioOutputControl::PlayChunk() noexcept
 		try {
 			const ScopeUnlock unlock(mutex);
 			nbytes = output->Play(data.data, data.size);
-			assert(nbytes > 0);
```

Comment:
It is needed. If you had crashes due to this, then you should fix the real bug.

Reply:
Just for caryaudio.  I changed the FifoOutputPlugin.cxx/Play(const void *chunk, size_t size), it may return 0. It's not a good change.